data_folder/work_preferences.yaml

```diff
@@ -48,4 +48,8 @@ location_blacklist:
 
 job_applicants_threshold:
   min_applicants: 0
-  max_applicants: 30
\ No newline at end of file
+  max_applicants: 30
+
+llm_model_type: openai
+llm_model: "gpt-4o-mini"
```

Comment:
No need to keep `llm_model_type` and `llm_model` here. They have been moved config.py

Reply:
Done